### ISSUE TITLE
Google drive link

### ISSUE THREAD

**Issue by TheMadScientiist** [VisDrone/DroneVehicle#34]
Hi thank you for your work! Are we still getting a Google drive link for the dataset?

**Comment by bozcani** [comment]
Any update?

**Comment by sheanps** [comment]
> Any update?

Nope, but I ended up using a simple process to help me pull from baidu. I used this YouTube tutorial https://youtu.be/8u8jx3Owuc8?si=nIuKCyLmnNEMECl5

**Comment by bozcani** [comment]
Thanks! Could you maybe upload to google drive or similar @TheMadScientiist ? I don't think the authors are maintaining the repo.

**Comment by sheanps** [comment]
> Thanks! Could you maybe upload to google drive or similar @TheMadScientiist ? I don't think the authors are maintaining the repo. 

I have to check my computer to see if I still have it, I remember downloading another dataset at the same time and can't remember which one I deleted.

**Comment by bozcani** [comment]
Thanks! Were you able to check @TheMadScientiist ? It would be super helpful if you could share it via google drive. bozcanilker@gmail.com if you like to add me

**Comment by sheanps** [comment]
> Thanks! Were you able to check @TheMadScientiist ? It would be super helpful if you could share it via google drive. bozcanilker@gmail.com if you like to add me

It seems I only have the training dataset, not including the validation and test dataset. If this isn't an urgent matter I can pull those down for you and upload them to the google drive.

**Comment by bozcani** [comment]
ok, not urgent. I can wait for val and test. Thanks!

**Comment by sheanps** [comment]
> ok, not urgent. I can wait for val and test. Thanks!

Check your gmail.

**Comment by bozcani** [comment]
> > ok, not urgent. I can wait for val and test. Thanks!
> 
> Check your gmail.

Got it! Thanks!

**Comment by shahla-ai** [comment]
hello @bozcani @TheMadScientiist
can i have the train dataset please ? 
my email is yamen.shahla1@gmail.com 
Thank you very much
best regards

**Comment by sheanps** [comment]
> hello @bozcani @TheMadScientiist
> 
> can i have the train dataset please ? 
> 
> my email is yamen.shahla1@gmail.com 
> 
> Thank you very much
> 
> best regards

@shahla-ai  Still need it?

**Comment by shahla-ai** [comment]
@TheMadScientiist Yes please

**Comment by sheanps** [comment]
> @TheMadScientiist Yes please

I gave you access 6 days ago, check your email.

**Comment by shahla-ai** [comment]
@TheMadScientiist 
i checked my email now , i didn't know what to do i didn't find any link 
can you please make it clear to me what should i do ?
Thank you

**Comment by shahla-ai** [comment]
@TheMadScientiist  
my bad , i found the correct email that you sent me more than a weak ago and i'm downloading the dataset
Thank you Bro
Best regards

**Comment by KyriakosChris** [comment]
hello @bozcani @TheMadScientiist and @shahla-ai. 
Can someone give me the train val and test Dataset? My email is sundayman98@gmail.com

Thank you very much!

**Comment by sheanps** [comment]
> hello @bozcani @TheMadScientiist and @shahla-ai. 
> 
> Can someone give me the train val and test Dataset? My email is sundayman98@gmail.com
> 
> 
> 
> Thank you very much!

Still need it @KyriakosChris ?

**Comment by KN-Zhang** [comment]
hello @bozcani @TheMadScientiist @shahla-ai!

I also need the datasets, whatever the train, val, or the test split.
Can someone help me? My email is zkn707196@gmail.com

Many thanks!

**Comment by KyriakosChris** [comment]
> > hello  @bozcani @TheMadScientiist and  @shahla-ai. 
> > Can someone give me the train val and test Dataset? My email is  [sundayman98@gmail.com ](mailto:sundayman98@gmail.com)
> > Thank you very much!
> 
> Still need it @KyriakosChris ?

I am sorry for not answering, I did not get any notification, I still need it, can you do it? 

Thanks a lot

**Comment by SanaaWaheeb** [comment]
@bozcani @TheMadScientiist @shahla-ai!
I also need the datasets. please can use send it on my email 
sanaa.books18@gmail.com

**Comment by GohYuJin** [comment]
@bozcani @TheMadScientiist @shahla-ai 
Hello, I would also like to have the datasets. It would be great if you could send it to my email trieightz@gmail.com

**Comment by sheanps** [comment]
@KyriakosChris @GohYuJin @KN-Zhang @SanaaWaheeb you should all have access to the Google drive folder. Please check the emails you provided.

**Comment by GohYuJin** [comment]
Yup I got it! Thank you @TheMadScientiist

**Comment by shemaamohamed** [comment]
@TheMadScientiist  Hello, I would also like to have the datasets. It would be great if you could send it to my email shemaaeldakhakhny@gmail.com

**Comment by kevinzxc1217** [comment]
@TheMadScientiist hello, I also want to the dataset . Grateful! my email is 
kevinzxc1217@gmail.com

**Comment by sheanps** [comment]
Done



On Tuesday, February 6, 2024, 1:23 AM, kevinzxc1217 ***@***.***> wrote:




@TheMadScientiist hello, I also want to the dataset . Grateful! my email is
***@***.***

**Comment by shemaamohamed** [comment]
Hello, I would also like to have the datasets. It would be great if you could send it to my email [ shemaaeldakhakhny@gmail.com]
@TheMadScientiist

**Comment by darkushin** [comment]
Hi @TheMadScientiist,
I would really appreciate it if you could share with me the link as well. My email is daniel.arkushin@gmail.com.
Thank you very much!

**Comment by RaiMunoz** [comment]
Hi @TheMadScientiist 
    I apologize for the hassle, but would it be possible to send share the link to my email at rmunoz00@citymail.cuny.edu? 

Thank you for your consideration!

**Comment by MiNareee** [comment]
Hello, your training dataset that you used in
the VisDrone/Drone Vehicle repository is very necessary. Can I get it please?
my email address milovanovanastya23@gmail.com
Thank you very much! I will be very grateful!

**Comment by MiNareee** [comment]
Hello, @TheMadScientiist, your training dataset that you used in
the VisDrone/Drone Vehicle repository is very necessary. Can I get it please?
my email address [milovanovanastya23@gmail.com](mailto:milovanovanastya23@gmail.com)
Thank you very much! I will be very grateful!

**Comment by sheanps** [comment]
@shemaamohamed @darkushin @RaiMunoz @MiNareee you should all have access to the dataset

**Comment by Jimmycedervalllamin** [comment]
Hi, @TheMadScientiist ! I would also love to get a link to the dataset, please! My mail is jimmycedervall@outlook.com

Thank you! :1st_place_medal:

**Comment by FawadAbbas12** [comment]
@TheMadScientiist can you provide access to dataset ?
my email is abk1391@gmail.com

**Comment by sheanps** [comment]
@FawadAbbas12 @Jimmycedervalllamin should be good to go

**Comment by sheanps** [comment]
For future individuals you should be able to access this link https://drive.google.com/drive/folders/1YsFm3cyO54d07wSvpsViHOGEYBuBUVWy

**Comment by Manjuphoenix** [comment]
Has anyone viewed the annotations before using and are the results consistent as mentioned in the paper??

After visualizing the annotations there is an issue, due to the white padding around the image. I think this would've occurred during the image registration process where the image has to be cropped due to the difference in FOV's of EO and IR camera as seen below.
![Screenshot from 2024-06-11 12-58-20](https://github.com/VisDrone/DroneVehicle/assets/56871357/0d5e4be7-249d-423f-9b8c-afd4bd4d914f)
![Screenshot from 2024-06-11 12-58-28](https://github.com/VisDrone/DroneVehicle/assets/56871357/0e7545b4-4716-4f54-b4cf-e99a6b2f5e19)
![Screenshot from 2024-06-11 12-58-38](https://github.com/VisDrone/DroneVehicle/assets/56871357/fa988aa8-4a84-4902-a3bd-eb1ce134a1b0)

If anyone has observed the same issue please let me know below.

**Comment by MMarvasti** [comment]
@TheMadScientiist Could you please provide access to the dataset?
my email is mojtaba.marvasti@gmail.com

**Comment by Yasir-ali-farrukh** [comment]
Thanks mate, this really helped !

**Comment by Yasir-ali-farrukh** [comment]
> For future individuals you should be able to access this link https://drive.google.com/drive/folders/1YsFm3cyO54d07wSvpsViHOGEYBuBUVWy

Thanks man, saved a lot of hassle !

**Comment by zhao06696-source** [comment]
> 你好[@bozcani](https://github.com/bozcani) [@TheMadScientiist](https://github.com/TheMadScientiist) 可以给我一份训练数据集吗？ 我的邮箱是zhao06696@gmail.com非常感谢， 此致
